src/main/java/net/neoforged/neoforge/event/AddPackFindersEvent.java

```diff
@@ -29,8 +29,8 @@ public AddPackFindersEvent(PackType packType, Consumer<RepositorySource> sources
      * Adds a new source to the list of pack finders.
      *
      * <p>Sources are processed in the order that they are added to the event.
-     * Use {@link Pack.Position#TOP} to add low priority packs,
-     * and {@link Pack.Position#BOTTOM} to add high priority packs.
+     * Use {@link Pack.Position#TOP} to add high priority packs,
+     * and {@link Pack.Position#BOTTOM} to add low priority packs.
```

Comment:
That's just what top/bottom mean -- `TOP` and `BOTTOM` are quite literally the top and bottom of the pack selection screen, so if something is at the top it'll be higher priority (that is, its resources will be preferred over lower packs). And yeah, the docs were wrong before but it didn't matter because due to _another_ bug _everything_ got shoved at `TOP` even if you specified `BOTTOM`.